src/ray/common/scheduling/cluster_resource_scheduler.h

```diff
@@ -251,15 +264,13 @@ class ClusterResourceScheduler {
   int64_t NumNodes();
 
   /// Convert a map of resources to a TaskRequest data structure.
-  void ResourceMapToTaskRequest(
-      const std::unordered_map<std::string, double> &resource_map,
-      TaskRequest *task_request);
+  TaskRequest ResourceMapToTaskRequest(
+      const std::unordered_map<std::string, double> &resource_map);
 
   /// Convert a map of resources to a TaskRequest data structure.
-  void ResourceMapToNodeResources(
+  NodeResources ResourceMapToNodeResources(
```

Comment:
This doesn't seem like it needs to be a class method. Please make it a helper function in the .cc file and do not expose it in the header file.